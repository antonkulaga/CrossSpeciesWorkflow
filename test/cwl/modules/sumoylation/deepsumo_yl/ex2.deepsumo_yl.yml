fastaFile:
  class: File
  location: ../data/ex2.fasta
threshold: 0.5
outputFilename: ex2.deepsumo_yl.out
  