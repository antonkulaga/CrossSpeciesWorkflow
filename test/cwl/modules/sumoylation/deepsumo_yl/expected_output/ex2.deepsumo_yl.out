Fragments name,Sequence,Probability value
>DAXX_HUMAN K56,SSSEPHGARGSSSSGGKKCYKLENEKLFEEFLE,0.68194508553
>DAXX_HUMAN K57,SSEPHGARGSSSSGGKKCYKLENEKLFEEFLEL,0.57694673538
>DAXX_HUMAN K75,YKLENEKLFEEFLELCKMQTADHPEVVPFLYNR,0.70670634508
>DAXX_HUMAN K122,CNILSRVLSRARSRPAKLYVYINELCTVLKAHS,0.81137257814
>DAXX_HUMAN K135,RPAKLYVYINELCTVLKAHSAKKKLNLAPAATT,0.53698110580
>DAXX_HUMAN K140,YVYINELCTVLKAHSAKKKLNLAPAATTSNEPS,0.64732760191
>DAXX_HUMAN K208,EQLLALYVAEIRRLQEKELDLSELDDPDSAYLQ,0.81294006109
>DAXX_HUMAN K229,SELDDPDSAYLQEARLKRKLIRLFGRLCELKDC,0.76021569967
>DAXX_HUMAN K231,LDDPDSAYLQEARLKRKLIRLFGRLCELKDCSS,0.52940106392
>DAXX_HUMAN K295,GPDTFPDYGDVLRAVEKAAARHSLGLPRQQLQL,0.62468290329
>DAXX_HUMAN K392,ISKYAMLQDKSEEGERKKRRARLQGTSSHSADT,0.86408561468
>DAXX_HUMAN K393,SKYAMLQDKSEEGERKKRRARLQGTSSHSADTP,0.91436034441
>DAXX_HUMAN K490,QEDDEEEDEEEEAAAGKDGDKSPMSSLQISNEK,0.54864758253
>DAXX_HUMAN K591,PLDTPSSVETDISSSRKQSEEPFTTVLENGAGM,0.55784231424
>DAXX_HUMAN K631,GGVSPHNWGDSGPPCKKSRKEKKQTGSGPLGNS,0.54648441076
>DAXX_HUMAN K634,SPHNWGDSGPPCKKSRKEKKQTGSGPLGNSYVE,0.88989788294
>DAXX_HUMAN K636,HNWGDSGPPCKKSRKEKKQTGSGPLGNSYVERQ,0.82382786274
>DAXX_HUMAN K658,GPLGNSYVERQRSVHEKNGKKICTLPSPPSPLA,0.63597232103
>DAXX_HUMAN K721,RLSQTPHSQPPRPGTCKTSVATQCDPEEIIVLS,0.63126605749
>RNF111_HUMANK76,VGNEFSHLCDDSQKQEKEMNGNQQEQEKSLVVR,0.51118212938
>RNF111_HUMANK93,EMNGNQQEQEKSLVVRKKRKSQQAGPSYVQNCV,0.73034286499
>RNF111_HUMANK94,MNGNQQEQEKSLVVRKKRKSQQAGPSYVQNCVK,0.69082456827
>RNF111_HUMANK110,KRKSQQAGPSYVQNCVKENQGILGLRQHLGTPS,0.50322091579
>RNF111_HUMANK159,SSLHFGDSDTVTSDEDKEVSVRHSQTILNAKSR,0.60961592197
>RNF111_HUMANK214,KRPCLHGSSLRRLPCRKRFVKNNSSQRTQKQKE,0.55882829428
>RNF111_HUMANK227,PCRKRFVKNNSSQRTQKQKERILMQRKKREVLA,0.78204470873
>RNF111_HUMANK229,RKRFVKNNSSQRTQKQKERILMQRKKREVLARR,0.54167723656
>RNF111_HUMANK246,ERILMQRKKREVLARRKYALLPSSSSSSENDLS,0.50063896179
>RNF111_HUMANK910,NRGASQGTIERCTYPHKYKKVTTDWFSQRKLHC,0.53969144821
>RNF111_HUMANK977,MHLFHQVCVDQWLITNKKCPICRVDIEAQLPSE,0.60776394606
>RNF111_HUMANK978,HLFHQVCVDQWLITNKKCPICRVDIEAQLPSES,0.51977944374
